config.go

```diff
@@ -121,43 +121,38 @@ func (c *CPMConfig) getHosts(networkLabel string) []string {
 	return nil
 }
 
-func (c *CPMConfig) getSdkDestination(forLanguage string) string {
-	// TODO: this is a bit of a hack
-	// Resolve this properly once we have a second language that can do off-chain generation or a language that supports both
-	if forLanguage == LANG_TYPESCRIPT {
-		if c.Defaults.OffChain == nil {
```

Comment:
the removal of this check now causes a null pointer exception in the following scenario
* remove the "off-chain" key from the `defaults` sections in `cpm.yaml`
* run `cpm generate ts -m somecontract.manifest.json`
```go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xa6d4f6]

goroutine 1 [running]:
main.(*CPMConfig).getSdkDestination(0xc00024ae10, {0xb40194, 0x2}, {0x0, 0x0})
        /home/erik/code/cpm/config.go:129 +0xf6
main.handleCliGenerate(0xc0001f2c40, {0xb40194, 0x2})
        /home/erik/code/cpm/main.go:416 +0x2e5
```